go/worker/sandbox/dockerutil/dockerutil.go

```diff
@@ -30,7 +30,7 @@ func ImageExists(client *docker.Client, name string) (bool, error) {
 func SafeKill(client *docker.Client, cid string) error {
 	container_insp, err := client.InspectContainer(cid)
 	if err != nil {
-		return fmt.Errorf("failed to get inspect docker container ID %v: ", cid, err)
+		return fmt.Errorf("failed to get inspect docker container ID %v: %v", cid, err)
```

Comment:
For Errorf, %w is better than %v, because then you can get a stack of errors that you could inspect at a higher level if you wanted to.

Most of the code uses %v because I didn't know about %w (or maybe it's newer?).  But we should use %w for this going forward.